integration_tests/src/payments_althea.rs

```diff
@@ -65,11 +73,28 @@ pub async fn run_althea_payments_test_scenario() {
 
     test_routes(namespaces.clone(), expected_routes);
 
+    // Exits need to have funds to request a registered client list, which is needed for proper setup
+    info!("Topup exits with funds");
+    let web3 = Web3::new("http://localhost:8545", WEB3_TIMEOUT);
+    let mut to_top_up = Vec::new();
+    for c in rita_identities.client_identities.clone() {
```

Comment:
this should probably be a utility function since we'll want to do it at the start of every test.

Reply:
done